src/deploy/index.js

```diff
@@ -75,8 +77,16 @@ module.exports = async (api, siteId, dir, opts) => {
       draft: opts.draft
     }
   })
+  if (opts.deployId === null) {
+    if(title) {
```

Comment:
Quick question: wouldn't be simpler to keep `title` in `cleanDeep()`?

Reply:
Do you mean under the `opts.deployId === null` condition? 
https://open-api.netlify.com/#operation/updateSiteDeploy doesn't accept `title` (unless I'm missing something), so didn't want to do it outside the condition.
We can do it inside the condition, but didn't see a reason to use the dependency for just one property.
I could go either way.

Reply:
Oh, makes sense! :+1: